test/test.js

```diff
@@ -23,8 +23,11 @@ test.cb('rebuild the guetzli binaries', t => {
 		});
 });
 
-test('return path to binary and verify that it is working', async t => {
-	t.true(await binCheck(guetzli, ['--version']));
+test.cb('return path to binary and verify that it is working', t => {
+	binCheck(guetzli, ['--version']).then(result => {
+		t.true(result);
+	});
+	t.end();
 });
```

Comment:
Woops, you're right. I thought I double-checked that. Addressed in #7, hopefully.